client/src/components/Stripe-button/stripe-button.js

```diff
@@ -0,0 +1,44 @@
+import React from "react"
+import StripeCheckout from "react-stripe-checkout"
+import axios from "axios"
+//stripe sees data in cents not dollars
```

Comment:
This can be a helpful note to yourself, but should be removed before commit. If you want to show something thats already elsewhere in docs you should link to it.